### ISSUE TITLE
import error with python2 package

### ISSUE THREAD

**Issue by abelboldu** [termux/termux-packages#41]
Hi,

I successfully installed a package with pip, but when I try to run it I get the following error:
```
$ ansible
Traceback (most recent call last):
 File "/data/data/com.termux/files/usr/bin/ansible", line 36, in <module>
 from ansible.runner import Runner
 File "/data/data/com.termux/files/usr/lib/python2.7/site-packages/ansible/runner/__init__.py", line 37, in <module>
 import ansible.inventory
 File "/data/data/com.termux/files/usr/lib/python2.7/site-packages/ansible/inventory/__init__.py", line 26, in <module>
 from ansible.inventory.ini import InventoryParser
 File "/data/data/com.termux/files/usr/lib/python2.7/site-packages/ansible/inventory/ini.py", line 21, in <module>
 from ansible.inventory.host import Host
 File "/data/data/com.termux/files/usr/lib/python2.7/site-packages/ansible/inventory/host.py", line 19, in <module>
 from ansible import utils
 File "/data/data/com.termux/files/usr/lib/python2.7/site-packages/ansible/utils/__init__.py", line 29, in <module>
 from ansible.utils.display_functions import *
 File "/data/data/com.termux/files/usr/lib/python2.7/site-packages/ansible/utils/display_functions.py", line 22, in <module>
 from ansible.callbacks import display
 File "/data/data/com.termux/files/usr/lib/python2.7/site-packages/ansible/callbacks.py", line 30, in <module>
 from ansible.module_utils import basic
 File "/data/data/com.termux/files/usr/lib/python2.7/site-packages/ansible/module_utils/basic.py", line 57, in <module>
 import syslog
ImportError: dlopen failed: cannot locate symbol "__android_log_print" referenced by "syslog.so"...
```

This link suggests to add some extra flags to the build:

https://stackoverflow.com/questions/4455941/undefined-reference-to-android-log-print

**Comment by abelboldu** [comment]
BTW, having a binary package for Ansible would be nice too!

**Comment by fornwall** [comment]
Hi, thanks a lot for reporting! The syslog module have been fixed in version `python2-2.7.10-4` of the package, which you can install with `apt update && apt upgrade`. Let me know if there is any other problem with installing ansible!

By the way: In Termux syslog is implemented as a wrapper around the Android logging system, use something like `logcat -s 'syslog:*'` to see what is output to syslog!

**Comment by bhupi8100** [comment]
Python2 hack insta.py error2 occured
What can I do
